src/questions/smart_contracts.yaml

```
@@ -0,0 +1,966 @@
+  answerKeys:
+    - C
+  subTopics:
+    - "miner-extractable-value"
+  difficultyLevel: "Low"
+  choices:
+    - content: Blocks
+      key: A
+    - content: Sender’s Wallet
+      key: B
+    - content: Mempool
+      key: C
+    - content: Miner’s Wallet
+      key: D
+- uuid: e9e60193-bd9e-4855-9a18-dc4cb2ea2508
+  type: MultipleChoice
+  content: What happens when mempool is overloaded with transactions?
+  hint: Overcrowded Mempool creates more time in successful transaction.
+  explanation: When the mempool becomes overloaded with transactions, it resutls in delayed confirmations of these transactions and increase in the gas fees as well.
+  answerKeys:
+    - A
+    - C
+  subTopics:
+    - "miner-extractable-value"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Delayed confirmation of transactions
+      key: A
+    - content: Reduction in backlog.
+      key: B
+    - content: Increase in transaction fees.
+      key: C
+    - content: Low traffic in the pool
+      key: D
+- uuid: 7c548058-2294-4443-9ee6-61e8525246e9
+  type: MultipleChoice
+  content: What users can do if there is an overload in the mempool?
+  hint: NoHint
+  explanation: When there is too much traffic in the mempool, users might switch to a lightning network which would lead to more transaction fees. Otherwise users can just wait to get a confirmation.
+  answerKeys:
+    - B
+    - C
+    - D
+  subTopics:
+    - "miner-extractable-value"
+  difficultyLevel: "High"
+  choices:
+    - content: Cancel the transaction
+      key: A
+    - content: Switch to lightning network
+      key: B
+    - content: Pay more transaction fees
+      key: C
+    - content: Wait for confirmation
+      key: D
+- uuid: db87beb8-09d7-42f4-8858-fa436bb01344
+  type: SingleChoice
+  content: How can miners tamper the blockchain?
+  hint: NoHint
+  explanation: Miners have access to the mempool. So, they can exclude a transaction, include a transaction or re-ordering any transaction in the pool.
+  answerKeys:
+    - D
+  subTopics:
+    - "miner-extractable-value"
+  difficultyLevel: "Low"
+  choices:
+    - content: Excluding transaction
+      key: A
+    - content: Including transaction
+      key: B
+    - content: Re-ordering transactions
+      key: C
+    - content: All of the above
+      key: D
+- uuid: 8aeae442-78fd-4e64-98e7-859b6231c143
+  type: SingleChoice
+  content: What does Miners Extractable Value refer to?
+  hint: Extra Income for miners
+  explanation: Miner Extractable Value is the extra income other than standard block reward and gas fees which the miners get by tampering with the transaction orders (front-running).
+  answerKeys:
+    - D
+  subTopics:
+    - "miner-extractable-value"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Standard block reward
+      key: A
+    - content: Gas fees
+      key: B
+    - content: Interest on large order
+      key: C
+    - content: None of the above
```

```suggestion
    - content: ENTER CORRECT OPTION HERE
```

This will help user know about MEV

---

DOne